src/main/resources/migration/common/V24__approved_dates.sql

```diff
@@ -0,0 +1,22 @@
+create table approved_dates_submission
+(
+    id                     serial
+        constraint approved_dates_request_pk primary key,
+    calculation_request_id integer                  NOT NULL references calculation_request (id),
+    prisoner_id            varchar(10)              not null,
```

Comment:
aren't some of the fields here below duplicated in the calculation request table? e.g prisonerId, booking Id?

could we piggy back from those values? or maybe faster to read if we stamp here as well

Reply:
They are duplicated yes. There will be instances when the approved dates for a prisoner could be 'removed', so having them also stored here will make that easier. It's a small amount of future proofing. There will also be instances where we want to look up just the approved dates without getting all of the calculation information (e.g on hub) so having it here will make that easier.